.github/dependabot.yml

```
@@ -0,0 +1,24 @@
+version: 2
```

NIP: I know it's not required to start `YAML documents from `---`, but it's suggested so, maybe let's stick to it:

```suggestion
---
version: 2
```